build.sbt

```diff
@@ -44,6 +44,7 @@ lazy val core = project
   .in(file("core"))
   .settings(
     name := "paradox",
+    crossScalaVersions := Seq(scalaVersion.value, "2.13.2"),
```

Comment:
Isn't cross-compiling the `core` module enough? This will even cross-compile the sbt plugin and the testkit.

Reply:
I thought this would cross compile the core since it's applied on `lazy val core`.
How do you cross compile the core?

Reply:
Ah, sorry. You're right. To the lazy me Github showed it looked like it was `inThisBuild`.
I'll have a closer look...